Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 
 name = "iomrascálaí"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
I'm fine with bumping it down to 0.1.x, but I thought time-keeping was a major feature to add. I'm not quite sure of to do semver well, though, so if you disagree I'm fine with it as well.